packages/indy-vdr/src/index.ts

```diff
@@ -1,3 +1,7 @@
+export * from './dids'
+export * from './error'
+export * from './pool'
```

Comment:
Do we want to export all these services / methods? I want to be more careful going forward in what we see as public api. I think seeing the specific things built on top of the pool service (so anoncreds registry, did resolver, did registrar) as public api means we can more easily make non breaking changes.

What do you think?

Reply:
Yes, maybe it's better to be more selective from the beginning. 

For the moment I only picked the `IndyDidSovResolver`, which is the only class (among the existing ones) I think it will be used externally once an IndyVdrModule or so is created.